Buy/Global.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Global.swift
+//  Buy
+//
+//  Created by Dima Bart on 2017-02-24.
+//  Copyright © 2017 Shopify Inc. All rights reserved.
+//
+
+import Foundation
+
+private final class BundleCoordinator: NSObject {
+    
+    static func frameworkBundle() -> Bundle {
+        return Bundle(for: self.classForCoder())
```

Comment:
You're right. Oddly enough, this is actually left over from previous experiments by accident (when it was previously an instance method). In this case, we can just use `self` instead. To take it further, we can just use `self` of `Global` and eliminate this `NSObject` subclass altogether.